### ISSUE TITLE
Wrong message logged from validation of broken links

### ISSUE THREAD

**Issue by krysal** [WordPress/openverse-api#725]
## Description

The validation function is assuming every link is from an image so it logs something like this:

```sh
openverse-api-web-1  | [2022-05-27 18:30:49,758 - catalog.api.utils.validate_images -  89][INFO] Deleting broken image with ID 4cba47fc-1ba6-43e1-adda-646bf7c4e0ae from results.
```

But when verifying the identifier in the database is not found in the `image` table but in `audio`.

```sql
SELECT * FROM image WHERE identifier = '4cba47fc-1ba6-43e1-adda-646bf7c4e0ae';
--- Returns 0 rows

SELECT * FROM audio WHERE identifier = '4cba47fc-1ba6-43e1-adda-646bf7c4e0ae';
--- Returns 1 row
```

The following utility function should be generalized to apply to more media types (in the immediate case, for `audio` too).

## Reproduction

1. Can be observed when running the tests for audio
```sh
just api-test -k audio_integration
```
2. See the logs with
```sh
just logs web
```
3. Confirm one of the identifiers running the previous queries.

## Resolution

- [ ] 🙋 I would be interested in resolving this bug.

**Comment by ritesh-pandey** [comment]
There are two tasks here.

1. Enhancing log 
Do we have a one to one mapping between ES index and media types? If yes, we can use ES index name in log for identifying database table.

2. Refactor `validate_images` module
Refactor validate_image module to include audio files as well. We can use `validate_media` as a better module name and make it more generic.

**Comment by Mishrasubha** [comment]
Hey @krysal I would like to take up this issue :)

**Comment by krysal** [comment]
@ritesh-pandey 
> Do we have a one to one mapping between ES index and media types?

That's correct, each media has its own ES index, and your suggestion of subdividing into two tasks sounds good.

---

@Mishrasubha Thanks for showing your interest. Please go ahead!
